### DISCUSSION TITLE
Potential 100 FPS Limit

### DISCUSSION THREAD

**Discussion by StovallFamily** [seerge/g-helper#3144] [Q&A]
Thank you for the great work! I have G-Helper on all my Asus devices and I have been using AutoTDP within G-helper on my Rog Ally X as I find it does a fantastic job. 

I was wondering though if it is possible to have a 100 FPS limit option as I use AFMF 2. When I lock my games to 60, AFMF 2 will go over 120 frequently causing screen tearing, therefore I like to lock my games to 50 FPS and allow AFMF 2 to take it to 100 FPS, but when using Auto TDP, it sees that the FPS never reaches 120 and tries to get it there by increasing the wattage. I understand that this is expected behavior and it is nice that Auto TDP takes generated frames into account, but I was wondering if it is possible to add a 100 FPS Limit. I have tried locking my games to 45 and using the 90 fps limit, but for whatever reason it just doesn't feel as good as locking the game to 50. 

If it's not possible, or can't be done right now, I completely understand. I'll make due with what's available to me right now. I figured I would just ask. Thanks again for all the work you put into this! It has been absolutely awesome to use!

**Comment by seerge** [comment]
@StovallFamily hello, i think you mix 2 separate (even tho connected) things. 
- FPS limit is a setting in AMD driver (you can change it in G-Helper OR in AMD Adrenaline app, as this is the same setting)
- Auto TDP mode that dynamically adjusts power limits based on FPS measurements and if they reach a limit

Can you check if FPS limit by itself (w/o any AutoTDP) is being followed by AMD drivers when you use AFMF2 in your game ? You can also set it in Adrenaline to verify.

**Reply by StovallFamily** [reply]
Where in the Adrenaline app would I change the limit?

**Reply by seerge** [reply]
@StovallFamily under Gaming -> Graphics

![Screenshot 2024-09-22 103622](https://github.com/user-attachments/assets/a36eae00-e08c-4a90-8b9c-38070ad31006)
